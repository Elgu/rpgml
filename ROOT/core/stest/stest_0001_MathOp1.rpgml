Function _mop( op, x )
{
  math.MathOp1 ret( op=op );
  x    -> ret.in;
  return  ret.out;
}

Output[ 3 ] c = [ counter(), counter()*2, counter()*3 ];
print( _mop( "sqr", counter() ) + "\n" );
print( _mop( "sqr", c ) ); print( "\n" );

exit( counter() >= 2 ); # 0, 1, 2
