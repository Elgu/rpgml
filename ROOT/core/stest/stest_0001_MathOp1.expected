0
[ 0, 0, 0 ]
1
[ 1, 4, 9 ]
4
[ 4, 16, 36 ]
